client/src/Component/MeetingComponent/MeetingComponent.js

```diff
@@ -0,0 +1,187 @@
+import React, { useState } from "react";
+import Grid from "@material-ui/core/Grid";
+import Icon from "@material-ui/core/Icon";
+import { makeStyles } from "@material-ui/core/styles";
+import Button from "@material-ui/core/Button";
+import { createMuiTheme } from "@material-ui/core/styles";
+import { grey, orange } from "@material-ui/core/colors";
+import { ThemeProvider } from "@material-ui/styles";
+import Paper from "@material-ui/core/Paper";
+import shadows from "@material-ui/core/styles/shadows";
+import Avatar from "@material-ui/core/Avatar";
+import Typography from "@material-ui/core/Typography";
+import GettingStartedButton from "../GettingStartedButton/GettingStartedButton";
+import Box from "@material-ui/core/Box";
+
+// import classNames from 'classnames/bind';
+// import styles from './submit-button.css';
+
+// let cx = classNames.bind(styles);
+
+const outerTheme = createMuiTheme({
+  palette: {
+    secondary: {
+      main: orange[500],
+    },
+    primary: {
+      main: grey[600],
+    },
+  },
+});
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+  },
+  meeting_box: {
+    background: theme.palette.text.default,
+    boxShadow: shadows[3],
+    paddingBottom: theme.spacing(3),
+  },
+  // tagBar:{
+  //   width: "100%",
+  //   height: "5px",
+  //   margin: "0px",
+  // },
+  orange_background: {
+    background: "orange",
+    width: "100%",
+    height: "5px",
+    margin: "0px",
+  },
+
+  blue_background: {
+    background: "blue",
+  },
+
+  green_background: {
+    background: "green",
+  },
+  meeting_header: {
+    width: "100%",
+    height: "30%",
+    textAlign: "end",
+  },
+
+  profile: {
+    padding: theme.spacing(3),
+  },
+  content: {
+    margin: "10px",
+  },
+  meeting_body: {
+    margin: "20px 30px 50px",
+  },
+  meeting_footer: {
+    width: "100%",
+    paddingTop: "10px",
+    borderTop: "1px solid rgb(236, 230, 230)",
+    fontWeight: "bold",
+  },
+  button: {
+    textAlign: "right",
+  },
+  profile_name: {
+    lineHeight: "2px",
+    verticalAlign: "bottom",
+    paddingLeft: "5px",
+  },
+}));
+
+const MeetingComponent = () => {
+  const [state] = useState({
+    meeting: [
+      { duration: 15, type: "One-on-One" },
+      { duration: 30, type: "One-on-One" },
+      { duration: 45, type: "One-on-One" },
+    ],
+  });
+
```

Comment:
If this never changes, you don't need to useState, you can just use a constant.
If if does change, you should use `const [meeting, setMeeting] = useState([{duration...}, {duration...}, {duration...}])`